portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,35 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body, html {
+  height: 100%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.bg {
+    background-image: url("/images/mountain.JPG");
+    filter: blur(4px);
+    -webkit-filter: blur(4px);
+    height: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
 }
+
+#home_content {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white;
+    font-family: 'Garamond';
+    Background-color: black;
+    border-style: solid;
+    padding: 20px;
+    text-align: center;
+}
+
+#home_content ul {
+    list-style-type: square;
+}
+
+#home_content li:hover {
+    text-decoration: underline;
+}
+
```

Comment:
Alphabetize this file, as per https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Alphabetize style file